mods/persistence/game/objects/items/weapons/storage.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/storage/Initialize()
+	if(persistent_id)
+		startswith = null
+	. = ..()
+
+// This is called at Initialize() to match an items storage capabilities to its current items, which is undesirable
+// for items which have been loaded.
+/obj/item/storage/make_exact_fit()
+	if(persistent_id)
+		return
+	. = ..()
+
+/obj/item/storage/internal/pockets/Initialize(mapload, slots, slot_size)
+	if(persistent_id)
+		slots = storage_slots
+		slot_size = max_w_class
+	. = ..()
+
+SAVED_VAR(/obj/item/storage, can_hold)
```

Comment:
I think can_hold and cant_hold are constants? You sure its worth saving them if they get repopulated on boot?

Reply:
Generally speaking they are. ``can_hold`` however changes when ``make_exact_fit`` is called, which we have to override for loaded objects. Because of this, anything which calls that must have ``can_hold`` save. The amount of storage items that actually do though is pretty low, so I'll probably set those individually.

``cant_hold`` doesn't change though, so I'll remove that from the saved vars